AppCenterDataStorage/AppCenterDataStorage/Internal/Util/MSDocumentUtils.h

```diff
@@ -76,6 +76,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (BOOL)isSerializableDocument:(Class)classType;
 
+/**
+ * Get serializable dictionary from a document.
+ *
+ * @param document The document to convert to a dictionary.
+ *
+ * @return NSDictionary instance if document is serializable; nil otherwise.
+ */
++ (NSDictionary *)getSerializableDictionaryFromDocument:(id<MSSerializableDocument>)document;
```

Comment:
You have `NS_ASSUME_NONNULL_BEGIN` macro, return values are implicitly annotated with `nonnull`. Even tho It should still work, it would be a good thing to add corresponding annotation explicitly.

Reply:
Got it. Is it standard to add the `nullable` annotation in places that aren't covered by the `NS_ASSUME_NONNULL_BEGIN` macro as well? Or is `nullable` the implicit assumption in those cases?

Reply:
@AndeNavya My understanding of the convention is by default adds `NS_ASSUME_NONNULL_BEGIN`, then explicitly declare `nullable` whenever required.